resolwe/flow/utils/__init__.py

```diff
@@ -76,3 +79,27 @@ def set_item(dict_or_model, key, value):
         except ValueError:
             set_item(d, k, val)
         return val
+
+
+def get_apps_tools():
+    """Get application tools and their paths.
+
+    Return dict of app names as keys and paths to tools directories
```

Comment:
Change to:
Return a dict with application names as keys and paths to tools' directories as values. Applications without tools are omitted.